db/migrations/20190709160237_create_flip_storage_tables.sql

```diff
@@ -189,7 +203,12 @@ COMMENT ON TABLE maker.flip_kicks IS E'@name flipKicksStorage';
 -- SQL in this section is executed when the migration is rolled back.
 DROP INDEX maker.flip_kicks_address_id_index;
 DROP INDEX maker.flip_kicks_kicks_index;
```

Comment:
Not introduced by this PR but I wonder why we need this index? I guess it's probably used in a join 🤔

Reply:
Nice catch! Currently it's not getting referenced by any other table, but my guess is we were using it at some point to get a `created` value for bids. Now we're getting `created` from `flip_kick` events instead of this storage table, so it seems safe to delete the index.